src/Authoring/WinRT.SourceGenerator/WinRTTypeWriter.cs

```diff
@@ -130,6 +130,7 @@ class TypeDeclaration
         public string DefaultInterface;
         public string StaticInterface;
         public bool IsSynthesizedInterface;
+        public bool IsComponentType;

```

Comment:
Doesn't have to be fixed in this PR, but just for additional info — I've tried to use the CsWinRT package from this PR (as of https://github.com/microsoft/CsWinRT/pull/1377/commits/2a7b7548b237597e32de9dcfcac0927bb6fc6ff3) and can confirm that https://github.com/microsoft/CsWinRT/issues/1369 still happens (same repro project).